tests/lib/rules/no-unsupported-features/es-syntax.js

```diff
@@ -2489,27 +2490,27 @@ ruleTester.run(
                     code: "obj.import(source)",
                     options: [{ version: "12.0.0" }],
                 },
-                {
+                ...["12.17.0", "13.2.0"].map(v => ({
```

Comment:
In both cases, I did this spread-from-map approach to keep the object DRY, but we could also just duplicate the `valid` and `invalid` cases.